doc/modules/model_evaluation.rst

```diff
@@ -1133,6 +1133,14 @@ are predicted. This is useful if you want to know how many top-scored-labels
 you have to predict in average without missing any true one. The best value
 of this metrics is thus the average number of true labels.
 
+.. note::
+
+    This definition of coverage error differs from the standard one, which describes
```

Comment:
refer "(Tsoumakas et al., 2010).

I think the description isn't helpful. Just say that "Our implementation's score is 1 greater than theirs. This extends it to handle the degenerate case in which an instance has 0 true labels."

I think it's worth repeating this note in the function's docstring.